frontend/src/pages/CreateNFTPage.jsx

```diff
@@ -55,104 +113,163 @@ const CreateNFT = ({ wallet }) => {
+                            </label>
+                            <input
+                                type="text"
+                                id="name"
+                                name="name"
+                                value={formData.name}
+                                onChange={handleChange}
+                                required
+                                className="block bg-white text-black px-6 py-3 rounded-full w-full max-w-md transition duration-300 box-shadow hover: 0 0 20px #ff00ff shadow-lg mx-auto"
+                            />
+                        </div>
+                        <div className="flex flex-col items-center">
+    <label
+        htmlFor="description"
+        className="block text-2xl font-bold mb-2 text-white"
+        style={{ textShadow: '2px 2px 10px #00ffff' }}
+    >
+        Description
+    </label>
+    <textarea
+        id="description"
+        name="description"
+        value={formData.description}
+        onChange={handleChange}
+        ref={textareaRef}
+        rows={1}
+        className="block bg-white text-black px-6 py-3 rounded-xl w-full max-w-md transition duration-300 box-shadow hover: 0 0 20px #ff00ff shadow-lg mx-auto"
+        style={{
+            resize: 'none',
+            overflowY: 'hidden',
+            height: 'auto',
+            borderRadius: '30px', // Control the border-radius to avoid the circle shape
+        }}
+    />
+</div>
+                        <div className="flex flex-col items-center">
+                            <label
+                                htmlFor="price"
+                                className="block text-2xl font-bold mb-2 text-white"
+                                style={{ textShadow: '2px 2px 10px #00ffff' }}
+                            >
+                                Price (ETH)
+                            </label>
+                            <input
+                                type="number"
+                                id="price"
+                                name="price"
+                                value={formData.price}
+                                onChange={handleChange}
+                                required
+                                step="0.01"
+                                className="block bg-white text-black px-6 py-3 rounded-full w-full max-w-md transition duration-300 box-shadow hover: 0 0 20px #ff00ff shadow-lg mx-auto"
+                            />
+                        </div>
+                        <div className="flex flex-col items-center">
+                            <label
+                                htmlFor="file"
+                                className="block text-2xl font-bold mb-2 text-white"
+                                style={{ textShadow: '2px 2px 10px #00ffff' }}
+                            >
+                                Upload File
+                            </label>
+                            <div
```

Comment:
Prefer that non-interactive, visible elements (such as <div>) that have click handlers use the role attribute
## you can use role attribute rather than using events on no semantic elements like this.
```html
Bad Practice

<div onClick={() => {}} />

Recommended

<button onClick={() => {}} className="foo" />
<div className="foo" onClick={() => {}} role="button" />
<input type="text" onClick={() =>

```